### ISSUE TITLE
Error when downloading on fedora 39

### ISSUE THREAD

**Issue by yodatak** [LykosAI/StabilityMatrix#612]
Hi thanks a lot but i got this issue with the appimage on Fedora what should i do ?
```
Unpacking resources
Unpacking resources
Error: System.TypeInitializationException: The type initializer for 'Delegates' threw an exception.
 ---> System.DllNotFoundException: Could not load /var/home/yodatak/Projets/public/AI/Data/Assets/Python310/lib/libpython3.10.so with flags RTLD_NOW | RTLD_GLOBAL: libcrypt.so.1: cannot open shared object file: No such file or directory
   at Python.Runtime.Platform.PosixLoader.Load(String dllToLoad)
   at Python.Runtime.Runtime.Delegates..cctor()
   --- End of inner exception stack trace ---
   at Python.Runtime.PythonEngine.<>c.<set_PythonHome>b__27_0()
   at Python.Runtime.Runtime.<>c__DisplayClass106_0.<TryUsingDll>b__0()
   at Python.Runtime.Runtime.TryUsingDll[T](Func`1 op)
   at Python.Runtime.Runtime.TryUsingDll(Action op)
   at Python.Runtime.PythonEngine.set_PythonHome(String value)
   at StabilityMatrix.Core.Python.PyRunner.Initialize()
   at StabilityMatrix.Avalonia.Helpers.UnixPrerequisiteHelper.InstallVirtualenvIfNecessary(IProgress`1 progress)
   at StabilityMatrix.Avalonia.Helpers.UnixPrerequisiteHelper.InstallPackageRequirements(List`1 prerequisites, IProgress`1 progress)
   at StabilityMatrix.Core.Models.PackageModification.SetupPrerequisitesStep.ExecuteAsync(IProgress`1 progress)
   at StabilityMatrix.Core.Models.PackageModification.PackageModificationRunner.ExecuteSteps(IReadOnlyList`1 steps)
   
   ```

**Comment by zpaldi** [comment]
@yodatak I faced the exactly the same issue and I fixed it the below way.

The issue caused by a missing RPM package dependency.

Install the missing dependency:
```bash
$ sudo dnf install libxcrypt-compat
```
TL;DR

I have found a solution here:
[Error while loading shared libraries: libcrypt.so.1](https://support.gurobi.com/hc/en-us/articles/17316546307217-Error-while-loading-shared-libraries-libcrypt-so-1)

The missing file located here:
```bash
$ sudo find /usr -name "libcrypt.so.1" 2>/dev/null
/usr/lib64/libcrypt.so.1
$
```
The following RPM package contains the missing file  on my system (Fedora 39) at the time I am writing:
```bash
$ rpm -qf /usr/lib64/libcrypt.so.1
libxcrypt-compat-4.4.36-2.fc39.x86_64
```
